infrastructure/scripts/provision/core.sh

```diff
@@ -12,20 +11,19 @@ source ${INFRASTRUCTURE_PATH}/scripts/lib/k8s.sh
 cd ${INFRASTRUCTURE_PATH}/scripts/
 wait-for-service-account
 
-
 echo "Deploying the Airy Core Platform with the ${AIRY_VERSION} image tag"
 
 if [[ -f ${INFRASTRUCTURE_PATH}/airy.yaml ]]; then
     yq eval '.global.appImageTag="'${AIRY_VERSION}'"' -i ${INFRASTRUCTURE_PATH}/airy.yaml
```

Comment:
I think you have to do the same as me here for NGROK or else when you enable ngrok via the env var but have it disabled in your airy.yaml it will be disabled

Reply:
ah no sorry that was just before we had the controller 👌